### DISCUSSION TITLE
How to spawn a Armorstand...

### DISCUSSION THREAD

**Discussion by xImNoxh** [Minestom/Minestom#968] [Help] [answered]
Hello, how can I spawn an armorstand. I also want to teleport it. Is this possible?
Kind regards

**Comment by ZakShearman** [comment, marked as answer]
You can spawn an armor stand using the following:
```java
Entity entity = new Entity(EntityType.ARMOR_STAND);
entity.setInstance(yourInstance, yourPosition);
entity.spawn();
```

You can then teleport an entity after it is spawned using the following:
```java
entity.teleport(newPos);
```

If you need some more information check out the Entity documentation here: https://wiki.minestom.net/feature/entities
